src/build_system.ml

```diff
@@ -1284,7 +1284,8 @@ let build_pred g = Memo.exec Pred.build_def g
 let build_deps =
   Dep.Set.parallel_iter ~f:(function
     | Alias a -> build_file (Alias.stamp_file a)
-    | File f -> build_file f
+    | File f
+    | Exists f -> build_file f
```

Comment:
This seems a bit off to me. Logically, we should just build `Exists f` if it doesn't exist, right? But without this, the `github759` test will fail.